RecoTracker/FinalTrackSelectors/interface/TfGraphDefWrapper.h

```diff
@@ -0,0 +1,16 @@
+#ifndef RecoTracker_FinalTrackSelectors_TfGraphDefWrapper_h
+#define RecoTracker_FinalTrackSelectors_TfGraphDefWrapper_h
+
+#include "PhysicsTools/TensorFlow/interface/TensorFlow.h"
+
+class TfGraphDefWrapper {
+public:
+  TfGraphDefWrapper(tensorflow::GraphDef*);
+
+  tensorflow::GraphDef* getGraphDef() const;
```

Comment:
> > And then, the tensorflow::session() doesn't work.
> 
> Right, so we're back in the interface of `tensorflow::createSession()` ([#32128 (comment)](https://github.com/cms-sw/cmssw/pull/32128#issuecomment) 726662792).

#32271 that changes the `tensorflow::createSession()` to take `const GraphDef *` has been merged now so now this can be changed to return pointer-to-`const`
```suggestion
  const tensorflow::GraphDef* getGraphDef() const;
```